src/guides/v2.3/frontend-dev-guide/templates/template-email.md

```diff
@@ -352,10 +354,12 @@ To set those values:
 
 The sales emails are configured to display all of the above values, if they're configured in the admin. If you want to add those values to other email templates, you can use the following variables:
 
-    {% raw %}{{var store.getFrontendName()}}{% endraw %}
-    {% raw %}{{var store_email}}{% endraw %}
-    {% raw %}{{var store_phone}}{% endraw %}
-    {% raw %}{{var store_hours}}{% endraw %}
+```text
```

Comment:
the block still needs to be enclosed within {% raw %} {% endraw %} tags to make Liquid parser to ignore the special symbols {{ }}

Reply:
fixed